CMakeLists.txt

```diff
@@ -6,6 +6,9 @@
 
 cmake_minimum_required(VERSION 3.0.2)
 
-project(software-factory VERSION 0.1.0)
+project("PELUX Software Factory"
```

Comment:
"PELUX Baseline Software Factory" would be more correct according to our nomenclature I believe.

Reply:
Indeed, but does the average reader know about the difference when reading the pelux.io docs? I can of course change it though.

Reply:
Don't make it too long because it needs to go into the top left corner in the template we're using now.

Reply:
The project we have on pelux.io develops a platform and a SWF that we call the "Baseline" right? The users of this will need to understand the difference between the Blueprint and a Deployment, and for that reason I think it makes sense to talk about the Baseline (which is a reference which includes a SWF Deployment etc). Everyone that reads the title doesn't need to know this, but IMO this wording is actually ambiguous and misleading.

Reply:
Whatever, I changed it now. I just wanted it to say something better than "software-factory".